docs/index.md

```diff
@@ -1,68 +1,51 @@
-# Checkly provider
-This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks, groups and snippets.
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "checkly Provider"
+subcategory: ""
+description: |-
+  
+---
 
-You can find a quick [step-by-step guide](https://www.checklyhq.com/docs/integrations/terraform/) in Checkly's documentation.
+# checkly Provider
```

Comment:
@ianaya89 I see this is generated. Can we still get Checkly capitalised?

Reply:
Not sure how, but will take a look!